### PR TITLE
introduce mvn wrapper

### PR DESCRIPTION
**Pull request by grafjo**

### CODE REVIEW

**Review by strieflin — CHANGES_REQUESTED** [review]
Including the wrapper scripts and jar means we distribute these artifacts. This means, we have to fulfill the license obligations (redistribute license and notice files of the original artifacts). Thus, please create a `3RD-PARTY-LICENSES ` folder in the repository root, add a folder for the Maven wrapper artifacts, and put the license and notice files there. See the client or service modules for examples. 

BTW; You can go ahead and make the required changes for castor (see carbynestack/castor#23) as well as a preparation for the review (FYI: @sbckr).

**Comment by grafjo** [comment]
> Including the wrapper scripts and jar means we distribute these artifacts. This means, we have to fulfill the license obligations (redistribute license and notice files of the original artifacts). Thus, please create a `3RD-PARTY-LICENSES ` folder in the repository root, add a folder for the Maven wrapper artifacts, and put the license and notice files there. See the client or service modules for examples.
> 
> BTW; You can go ahead and make the required changes for castor (see [carbynestack/castor#23](https://github.com/carbynestack/castor/pull/23)) as well as a preparation for the review (FYI: @sbckr).

I added the  `3RD-PARTY-LICENSES ` part to this repo and also updated [carbynestack/castor#23]

File: mvnw [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+    echo "$(tr -s '\n' ' ' < "$1")"
+  fi
+}
+
+BASE_DIR=`find_maven_basedir "$(pwd)"`
+if [ -z "$BASE_DIR" ]; then
+  exit 1;
+fi
+
+##########################################################################################
```
Comment (strieflin) [inline mvnw:206]: I would suggest to remove `maven-wrapper.jar` from the commit. It will be downloaded by the script in case it is not available.

Comment (grafjo) [reply mvnw:206]: good point - I removed the jar

**Review by strieflin — CHANGES_REQUESTED** [review]
Please remove `sbom.xml`. See carbynestack/castor#23 for a rationale.

**Comment by Lila84** [comment]
> Please remove `sbom.xml`. See [carbynestack/castor#23](https://github.com/carbynestack/castor/pull/23) for a rationale.

Did that.

**Review by strieflin — APPROVED** [review]
LGTM

**Pull request merged**